blocks/hooks/test/anchor.js

```diff
@@ -58,7 +58,7 @@ describe( 'anchor', () => {
 
 		it( 'should do nothing if the block settings do not define anchor support', () => {
 			const attributes = { anchor: 'foo' };
-			const extraProps = addSaveProps( blockSettings, attributes );
+			const extraProps = addSaveProps( {}, blockSettings, attributes );
```

Comment:
Good catch, this is an issue with testing only for non-existence. It is very easy to satisfy 😃